acceptance_tests/features/inbox_internal.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Internal inbox
+  As an internal user
+  I need to view all inbox messages
+  So that I can work on the messages as required
+
+  Background: Internal user is already signed in
+    Given the internal user is already signed in
+
+  Scenario: If there are no messages the user will be informed of this.
+    Given the user has access to secure messaging
+    When they navigate to the inbox messages
+    Then they are informed that there are no messages
+
+  Scenario: User is able to view all Inbox messages.
+    Given the user has access to secure messaging
+    And the secure message database is populated with messages
```

Comment:
I think the BDD scenarios should be written in a language the user knows. The user wouldn't be talking about messages being populated in the database I guess. More user friendly language might be something like

```
Given the respondent has got messages in their inbox (Implied access to secure messaging)
When they navigate to the inbox messages
Then they are able to view all received messages 
```